fastai2/callback/hook.py

```diff
@@ -146,7 +146,7 @@ def layer_info(learn):
     def _track(m, i, o):
         return (m.__class__.__name__,)+total_params(m)+(apply(lambda x:x.shape, o),)
     layers = [m for m in flatten_model(learn.model)]
-    xb,_ = learn.dbunch.train_dl.one_batch()
+    xb,*_ = learn.dbunch.train_dl.one_batch()
```

Comment:
Batches are always a tuple (of maybe nested tuples) so this should not be necessary.

Reply:
If the tuple len > 2 it gives this error:
![image](https://user-images.githubusercontent.com/25873380/72281795-21694300-3633-11ea-9829-376a880a81fd.png)

Reply:
Oh, it's actually something different, we need to cut the tuple at the right place. Let's merge this part as is and I'll follow up to add that part.